src/models.js

```diff
@@ -446,6 +446,11 @@ async function encoderForward(self, model_inputs) {
     for (let key of self.session.inputNames) {
         encoderFeeds[key] = model_inputs[key];
     }
+    if (self.session.inputNames.includes('token_type_ids')) {
+        const token_type_ids = model_inputs.attention_mask.clone();
+        token_type_ids.data.fill(0n);
+        encoderFeeds.token_type_ids = token_type_ids
+    }
```

Comment:
To avoid overwriting existing `token_type_ids`, you should add a check here:

```suggestion
    if (!encoderFeeds.token_type_ids && self.session.inputNames.includes('token_type_ids')) {
        add_token_types(encoderFeeds);
    }
```

You can import the `add_token_types` function from [tokenizers.js](https://github.com/xenova/transformers.js/blob/b8719b12dd7b61f99340a7d27e32f105538daf2e/src/tokenizers.js#L2581)